versioned_docs/version-23.10/releases/centreon-os.mdx

```diff
@@ -432,6 +432,23 @@ Release date: `October 30, 2023`
 
 ## Centreon Collect
 
+### 23.10.9
+
+Release date: `July 17, 2024`
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [Broker] If a metric can't be inserted in the database, the metric is dropped with an error log but Broker continues to work, and others metrics are still inserted.
+- [Broker] Fixed an issue while trying to Connect to MySQL 8.0.35.]
+- [Broker] There was two errors concerning the gnutls version check:
+1. We wanted that the gnutls library on the computer is at least the same as the one used to compile broker.
+2. The packaging required a very old version of gnutls 3.3.29
+
+Now, we expect that the version of the gnutls library is at least 3.6.0. And in the packaging, broker will be installed only if a such gnutls library can be installed.
```

Comment:
```suggestion
- [Broker] Fixed an issue that occurred when inserting too large metrics into the database, which resulted in an error log.
- [Broker] There were two errors concerning the gnutls version check:
1. The gnutls library on Centreon had to be at least the same version as the one used to compile Broker. Now, Broker checks that the version of the gnutls library is at least 3.6.0.
2. The packaging required a very old version of gnutls.
Now, when Broker is installed we check that the version of gnutls is at least 3.6.0. If the version is older, it will be updated.
- [Broker] Added support for the default connection method to MySQL 8.0.35.
```